packages/documentation-site/patternfly-docs/content/design-guidelines/components/table/table.md

```diff
@@ -22,6 +22,10 @@ The elements mentioned below are similar for a table with compact or default spa
 ## Usage
 When using tables you need to consider the structure of the data you want to display and organize that information into columns. Columns will typically have column headers. Every row within a table must have a consistent format. If the table row includes actions, they should always be placed in the rightmost column(s).
 
+Table toolbars should only contain actions that directly impact the table. Any actions that are related to the overall page and its context (or impact the general page but not the table directly) should be placed outside of the table, at the top right of the page at the header level. This may include actions that edit the page header, actions that launch related windows, and so on.
+
+![UI screen with action button placed outside of the table.](./img/page-action.png)
```

Comment:
I would edit this screen to have a blue button in the table toolbar, and then a secondary button styling for the top right page wide actions. Saying this because I can see a situation where people get confused and think primary buttons should be on the top right of the page,  when in reality that's only if there's nothing at the table toolbar level (which is rare)

Reply:
good point -- lmk what you think about the image I added and if any tweaks would make it more helpful!

Reply:
No, that totally makes sense! I made some updates -- lmk what you think

Reply:
Ok this is nitpicky but I would remove the kebab at the header of the table (which tbh idk why that's even part of the symbol since that's not a thing haha) and I would also make the column rows white - cause the reason they're grey was because of the table striping, but since there are only 2 rows, they should either be the same color or one should be white and white grey - so I vote that they should both be white haha otherwise looks good!